packages/dx-chart-core/src/utils/series.js

```diff
@@ -1,15 +1,7 @@
 import { area } from 'd3-shape';
 import { dArea, dLine, dSpline } from '../plugins/series/computeds';
 
-// Based on https://en.wikipedia.org/wiki/Uniform_norm
-const getUniformDistance = (dx, dy, rx, ry) => Math.max(
-  Math.abs(dx) / rx, Math.abs(dy) / ry,
-);
-
-// Based on https://en.wikipedia.org/wiki/Norm_(mathematics)#Euclidean_norm
-const getEuclideanDistance = (dx, dy, rx, ry) => Math.sqrt(
-  (dx * dx) / (rx * rx) + (dy * dy) / (ry * ry),
-);
+const getLength = (dx, dy) => Math.sqrt(dx * dx + dy * dy);
```

Comment:
Maybe we should rename this function to `getDistance` for example? Because of the `length` word used for arrays and string and return a number of values, but in current place, we would like to see a distance between two points.

`getRadius`, `getHypotenuse`

Reply:
Renamed.